lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1922,6 +1922,24 @@ Value MemOp::getPortNamed(StringAttr name) {
   return Value();
 }
 
+static Value trackValueThroughAnoymousWires(Value value) {
+  auto wire = value.getDefiningOp<WireOp>();
+  // If the value is not a wire, return the value.
+  if (!wire)
+    return value;
+
+  // If the wire has a symbol, return the wire.
+  if (wire.inner_sym() && !wire.inner_sym().getValue().empty())
+    return value;
+
+  for (auto *c : value.getUsers())
+    if (auto connect = dyn_cast<FConnectLike>(c))
```

Comment:
Are we out of last-connect semantics by this point?  If not, there might be multiple connects and this only checks the first one.

Reply:
If nothing else, we should be checking that there is only one connect.

Reply:
`getModuleScopeDriver` assumes sigle connection so I'll follow-up to add assertions there.